R/conttables.b.R

```diff
@@ -602,11 +602,15 @@ contTablesClass <- R6::R6Class(
             postHoc    <- self$results$postHoc
 
             subNamesPh    <- c('[resU]', '[resP]', '[resS]', '[resA]')
-            subTitlesPh   <- c(.('Unstandardized'), .('Pearson'), .('Standardized'), .('Adjusted'))
+            subTitlesPh   <- c(.('Unstandardized Residuals'), .('Pearson Residuals'), .('Standardized Residuals'), .('Deviance Residuals'))
             visiblePh     <- c('(resU)', '(resP)', '(resS)', '(resA)')
             typesPh       <- c('number', 'number', 'number', 'number')
             formatsPh     <- c('', '', '', '')
 
+            # For post-hoc tests
+            if (self$options$get('resA'))
+                postHoc$setNote('notetodeviance', .('Deviance Residuals are adjusted residuals from a Poisson GLM.'), init=TRUE)
```

Comment:
Sentence case please:
```suggestion
                postHoc$setNote('notetodeviance', .('Deviance residuals are adjusted residuals from a Poisson GLM.'), init=TRUE)
```